js/PageLevelProgressView.js

```diff
@@ -13,6 +14,7 @@ define([
         initialize: function() {
             this.listenTo(Adapt, 'remove', this.remove);
             this.render();
+            this.addChildren();
         },
 
         scrollToPageElement: function(event) {
```

Comment:
Need to return early if button is ‘disabled’.

Reply:
Done